src/app/components/pages/wallets/address-detail/wallet-detail.component.ts

```diff
@@ -94,7 +96,11 @@ export class WalletDetailComponent implements OnInit {
   deleteWallet() {
     const dialogRef = this.dialog.open(ConfirmationComponent, {
       width: '500px',
-      data: { text: `${this.deleteConfirmation1} "${this.wallet.label}" ${this.deleteConfirmation2}` }
+      data: {
+        text: `${this.deleteConfirmation1} "${this.wallet.label}" ${this.deleteConfirmation2}`,
+        isCheckboxDisplayed: true,
```

Comment:
Please change the name of the property from `isCheckboxDisplayed` to `displayCheckbox`.

Reply:
Fixed